### PR TITLE
Add `thread_group` to log payload

### PR DESCRIPTION
**Pull request by ofek**
### Description of the Change

Follow-up to https://github.com/DataDog/jmeter-datadog-backend-listener/pull/17

### CODE REVIEW

File: src/main/java/org/datadog/jmeter/plugins/util/CommonUtils.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package org.datadog.jmeter.plugins.util;
+
+public class CommonUtils {
+
+    public static String parseThreadGroup(String threadName) {
```
Comment (hithwen) [inline src/main/java/org/datadog/jmeter/plugins/util/CommonUtils.java:5]: can threadname be null?

Comment (hithwen) [inline src/main/java/org/datadog/jmeter/plugins/util/CommonUtils.java:5]:
```suggestion
    public static String parseThreadGroup(@NonNull  String threadName) {
```
can threadname be null?

**Review by ofek — COMMENTED** [review]
No it cannot

**Review by FlorianVeaux — APPROVED** [review]
Looks good to me, but have a look at Julia's comment 👍

**Comment by ofek** [comment]
I reverted that as it seems we'd need a third party package https://stackoverflow.com/a/35896657/5854007

**Review by FlorianVeaux — APPROVED** [review]

**Comment by clefego** [comment]
How do I know the default tag list? How to include a custom tag with "Response message" field values? In the Backend Listener list there is a customTags field, but I don't know how to pass the values to a custom tag.

**Pull request merged**